Packages/com.unity.multiplayer.samples.coop/Utilities/Net/ClientAuthority/ClientNetworkTransform.cs

```diff
@@ -0,0 +1,47 @@
+using Unity.Netcode.Components;
+using Unity.Netcode;
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.Utilities.ClientAuthority
+{
+    // TODO change to owner netvar instead of RPC based
```

Comment:
```suggestion
    // TODO inherit from `NetworkBehaviour` instead of `NetworkTransform` to cut direct relationship between two
    // TODO change to owner netvar instead of RPC based
```